ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Background.hs

```diff
@@ -70,7 +71,7 @@ import           Ouroboros.Consensus.Storage.ChainDB.API (BlockComponent (..))
 import           Ouroboros.Consensus.Storage.ChainDB.Impl.ChainSel
                      (addBlockSync)
 import           Ouroboros.Consensus.Storage.ChainDB.Impl.LgrDB
-                     (LgrDbSerialiseConstraints)
+                     (LgrDbSerialiseConstraints, LgrDB)
```

Comment:
Does `stylish-haskell` catch this? I anticipate that upper case comes before lower case in its sorting.

Reply:
yes, stylish uses one ordering function, I'm not sure how it treats upper vs lower case, would have to check